.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+on: [push, pull_request]
```

Comment:
@junha1 Does the "Recover CI" commit have the same contents as before?
If it has the same contents as before, I'll not read the commit in detail.

Reply:
It is same except `1.45.2` -> `1.46.0`